great_expectations/rule_based_profiler/parameter_builder/value_set_multi_batch_parameter_builder.py

```diff
@@ -156,6 +156,12 @@ def _get_unique_values_from_nested_collection_of_sets(
     """
 
     flattened: List[Set[Any]] = list(itertools.chain.from_iterable(collection))
+
+    """
+    In multi-batch data analysis, values can be empty and missin, resulting in "None" added to set.  However, due to
```

Comment:
@alexsherstinsky Can you also add `This ParameterBuilder filters null values from the unique value_set.` to the class docstring?